src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      calculator.result += value;
+
+      return calculator;
+    },
+
+    subtract(value) {
+      calculator.result -= value;
+
+      return calculator;
+    },
+
+    divide(value) {
+      calculator.result /= value;
+
+      return calculator;
+    },
+
+    multiply(value) {
+      calculator.result *= value;
+
+      return calculator;
+    },
+
+    reset() {
+      calculator.result = 0;
+
+      return calculator;
+    },
+
+    operate(callback, value) {
```

Comment:
```suggestion
    operate(callback, value) {
```
value is a bad name for the variable